mutable_tree.go

```diff
@@ -491,6 +506,18 @@ func (tree *MutableTree) GetVersioned(key []byte, version int64) (
 	return -1, nil
 }
 
+// GetVersionedFast gets the value at the specified key and version. The returned value must not be
+// modified, since it may point to data stored within IAVL. GetVersionedFast utilizes a more performant
+// strategy for retrieving the value than GetVersioned but falls back to regular strategy if fails.
+func (tree *MutableTree) GetVersionedFast(key []byte, version int64) []byte {
```

Comment:
Similar reasoning to [this](https://github.com/osmosis-labs/iavl/pull/5/files#r785243578)